src/HIE/Bios/Cradle.hs

```diff
@@ -566,8 +573,8 @@ cabalCradle l cs wdir mc projectFile
 -- to the custom ghc wrapper via 'hie_bios_ghc' environment variable which
 -- the custom ghc wrapper may use as a fallback if it can not respond to certain
 -- queries, such as ghc version or location of the libdir.
-cabalProcess :: LogAction IO (WithSeverity Log) -> CradleProjectConfig -> FilePath -> String -> [String] -> CradleLoadResultT IO CreateProcess
-cabalProcess l cabalProject workDir command args = do
+cabalProcess :: CompilationProgressReporter -> LogAction IO (WithSeverity Log) -> CradleProjectConfig -> FilePath -> String -> [String] -> CradleLoadResultT IO CreateProcess
+cabalProcess _ l cabalProject workDir command args = do
```

Comment:
Why unused variable?

Reply:
Good point, not sure how I missed that. Fixed here: https://github.com/haskell/hie-bios/commit/92a2359ab0c13e9f6106d2e4db5c70e29b0591cd